doc/devel/contribute.rst

```diff
@@ -183,6 +183,23 @@ please cite us following the :doc:`/project/citing` guidelines.
 If you have developed an extension to Matplotlib, please consider adding it to our
 `third party package <https://github.com/matplotlib/mpl-third-party>`_  list.
 
+
+.. _generative_ai:
+
+
+
+Use of Generative AI
+--------------------
```

Comment:
Also, I suggest to name this “Rules for Generative AI” or “Limitations for Generative AI”, because “Use …” sounds too positive and we mostly want to emphasize “do not use”

Reply:
done on both accounts.